/**:
  ros__parameters:
    namespace: ""
    printf_string: "ROS 3のテンプレートも欲しい："
    printf_hz: 10.0  #double